argo-cd-apps/base/member/infra-deployments/image-controller/image-controller.yaml

```diff
@@ -12,9 +12,11 @@ spec:
               values:
                 sourceRoot: components/image-controller
                 environment: staging
-                clusterDir: ""
+                clusterDir: base
```

Comment:
Changing the default `clusterDir` affect both prod and staging. You moved all the files from `components/image-controller/production` to `components/image-controller/production/base` which is fine but you need to do the same for staging, i.e. move `components/image-controller/staging` into `components/image-controller/staging/base`

Reply:
will do